### ISSUE TITLE
[enhancement] Use array in @post hooks

### ISSUE THREAD

**Issue by RyannGalea** [szokodiakos/typegoose#313]
I have instances where I want to achieve the same thing using different hooks but just end up having to duplicate the code, would it be possible or be a good idea to allow an array of hooks to do the same thing?

Currently:

          @Post<Contact>('save', (data) => addNotePost(data, Entities.CONTACT))
          @Post<Contact>('findOneAndUpdate', (data) => addNotePost(data, Entities.CONTACT))

Enhancement:

          @Post<Contact>(['save', 'findOneAndUpdate'], (data) => addNotePost(data, 
           Entities.CONTACT))

**Comment by hasezoey** [comment]
i think this is not within the scope of typegoose, but you could make a simple function like below:

```ts
function hookwrapper(next: () => void) {
    return addNotePost(next, Entities.CONTACT);
}

@Post<Contact>("save", hookwrapper)
@Post<Contact>("findOneAndUpdate", hookwrapper)
```
Note: this should work, but somehow i found that "save" and many others are not declared right (only "count" and "insertMany" are declared)

**Comment by hasezoey** [comment]
@Ben305 this can be closed because it is included in v6.0.0
